### PR TITLE
Changed the GPG key location

### PR DESCRIPTION
**Pull request by 0rnela**
https://github.com/SUSE/spacewalk/issues/24347

- [x] - master
- [x] - manager 5.0 https://github.com/uyuni-project/uyuni-docs/pull/3171

### CODE REVIEW

**Comment by keichwa** [comment]
@0rnela I guess, the user will create the GPG key on the container host.  If that's the case, the user will have to copy it with `mgradm gpg add` to the container, probably during the first procedure of this file.  That's how I understand Michele's comment in the referenced spacewalk issue.

**Review by keichwa — CHANGES_REQUESTED** [review]
Please see my comment in the "Conversation" tab.

File: modules/client-configuration/pages/autoinst-owngpgkey.adoc [hunk 1]
```diff
@@ -40,6 +39,12 @@ You must add our own key to this file, so it can be used to check the packages.
 To access a shell inside the Server container run [literal]``mgrctl term`` on the container host.
 ====
 
+[IMPORTANT]
+====
+To copy the GPG keys the container use [command]``mgradm gpg add``.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-owngpgkey.adoc:44]:
```suggestion
To copy GPG keys to the container use [command]``mgradm gpg add``.
```

**Review by keichwa — COMMENTED** [review]
I unfortunately do not know whether the `cp` step inside of the container is superfluous.  Maybe, `mgradm gpg add` can copy it to the final destination in one go.

File: modules/client-configuration/pages/autoinst-owngpgkey.adoc [hunk 2]
```diff
@@ -40,6 +39,12 @@ You must add our own key to this file, so it can be used to check the packages.
 To access a shell inside the Server container run [literal]``mgrctl term`` on the container host.
 ====
 
+[IMPORTANT]
+====
+To copy GPG keys to the container use [command]``mgradm gpg add``.
+====
+
+
 .Procedure: Add a GPG key to the initial RAM disk
```
Comment (mbussolotto) [inline modules/client-configuration/pages/autoinst-owngpgkey.adoc:48]: This part can be removed. The user just need to run `mgradm gpg add` then `mgradm restart` .
- `mgradm gpg add` command copies the GPG keys and add it to the customer keyring 
- `mgradm restart` create a unique keyring using the suse and the customer keyring

All the application should use the unique keyring mentioned ^^.

**Review by mbussolotto — COMMENTED** [review]
I didn't test it (maybe @aaannz can provide some feedback) but it should work as described in the comment...if not it's not a doc issue but a bug :)

File: modules/client-configuration/pages/autoinst-owngpgkey.adoc (hunk 2 above)
Comment (0rnela) [reply modules/client-configuration/pages/autoinst-owngpgkey.adoc]: Thanks, Implemented. Please see the attached PDF.

**Comment by 0rnela** [comment]
[newGPGprocedure.pdf](https://github.com/user-attachments/files/16144466/newGPGprocedure.pdf)

**Review by mbussolotto — APPROVED** [review]
LGTM!

**Review by keichwa — APPROVED** [review]
Thank you both!

**Comment by 0rnela** [comment]
Thank you all.

**Pull request merged**